City-of-Helsinki/hitas
# Apartment form validation

# Hitas Pull Request

# Description

- Refactor and add extensive frontend validation for apartment create/edit form.
- Small improvements and fixes
- Bump dependencies

## Pull request checklist

Check the boxes for each DoD item that has been completed:

- **Testing**
- **Database**
- **Documentation**

## Test plan

- Try to break apartment create form
- See if there is any wrong/missing validation rules in apartment create form

## Tickets

This pull request resolves all or part of the following ticket(s): HT-237
